angular/projects/spark-angular/src/lib/components/sprk-stack/sprk-stack.stories.ts

```diff
@@ -371,25 +333,27 @@ export const stackSplitLayoutMixed = () => ({
     <sprk-stack splitAt="tiny">
       <div
         sprkStackItem
-        class="sprk-o-Stack__item--fourth@xs sprk-u-AbsoluteCenter"
+        class="sprk-o-Stack sprk-o-Stack__item--fourth@xs sprk-o-Stack--center-all"
       >
-        <p class="sprk-b-TypeBodyOne">
+        <p sprkStackItem class="sprk-b-TypeBodyOne">
           fourth
         </p>
       </div>
+
       <div
         sprkStackItem
-        class="sprk-o-Stack__item--half@xs sprk-u-AbsoluteCenter"
+        class="sprk-o-Stack sprk-o-Stack__item--half@xs sprk-o-Stack--center-all"
       >
-        <p class="sprk-b-TypeBodyOne">
+        <p sprkStackItem class="sprk-b-TypeBodyOne">
           half
         </p>
       </div>
+
       <div
         sprkStackItem
-        class="sprk-o-Stack__item--fourth@xs sprk-u-AbsoluteCenter"
+        class="sprk-o-Stack sprk-o-Stack__item--fourth@xs sprk-o-Stack--center-all"
       >
-        <p class="sprk-b-TypeBodyOne">
+        <p sprkStackItem class="sprk-b-TypeBodyOne">
```

Comment:
We'll update these examples to use the sprkText directive in a future issue. I enhanced a lot of stuff in this PR and was pulled to work on other stuff so I didn't finish enhancing these paragraphs to use the sprkText directive as well. If i get time can add it in follow up PR.